cmd/thanos/receive.go

```diff
@@ -306,11 +306,14 @@ func runReceive(
 			return errors.Wrap(err, "setup gRPC server")
 		}
 
-		mts := store.NewMultiTSDBStore(
+		mts := store.NewProxyStore(
 			logger,
 			reg,
+			dbs.TSDBLocalClients,
 			comp,
-			dbs.TSDBStores,
+			labels.Labels{},
+			10*time.Second,
```

Comment:
Thanks for catching this, I actually wanted to change this and disable timeout (make it `0`) since there's no network call involved, so I believe it should be fine.

Reply:
So we're not changing this?

Reply:
I'm about to push, sorry for the delays